app/models/user_auth_token.rb

```diff
@@ -30,23 +30,23 @@ def self.log(info)
     end
   end
 
-  # Returns the login location as it will be used by the the system to detect
-  # suspicious login.
-  #
-  # This should not be very specific because small variations in location
-  # (i.e. changes of network, small trips, etc) will be detected as suspicious
-  # logins.
-  #
-  # On the other hand, if this is too broad it will not report any suspicious
-  # logins at all.
-  #
-  # For example, let's choose the country as the only component in login
-  # locations. In general, this should be a pretty good choce with the
-  # exception that for users from huge countries it might not be specific
-  # enoguh. For US users where the real user and the malicious one could
-  # happen to live both in USA, this will not detect any suspicious activity.
+  RAD_PER_DEG = Math::PI / 180
+  EARTH_RADIUS_KM = 6371 # kilometers
+
   def self.login_location(ip)
-    DiscourseIpInfo.get(ip)[:country]
+    ipinfo = DiscourseIpInfo.get(ip)
+
```

Comment:
Why the empty line?

Reply:
I just wanted to make the return value more obvious.